addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -279,6 +279,19 @@ registerModel({
             }
             return this.isAddingChat;
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeMobileAddItemHeaderInputView() {
+            if (
+                this.messaging.device.isMobile &&
+                (this.isAddingChannel || this.isAddingChat)
```

Comment:
Shouldn't there be 2 different `InputView` fields, one for each case? And also then boolean are redundant similar to the rest.

Reply:
In desktop it's 2 different locations, but in mobile the inputs are in different tabs, so only one occurence.
I prefer keeping logic that way for this PR, if we want to make adjustment, it's safer after this PR or at the very end.

Reply:
They are fundamentally 2 different inputs with 2 different purposes, it's not because they "display at the same place" that they are the same. The sooner we split them, the better. We should not have boolean variable to control the view record, its existence should suffice. With the current approach it's not possible to define a proper inverse "... owner as ..." and base conditions on it if there are 2 conflicting use cases for the view.